### DISCUSSION TITLE
New release...

### DISCUSSION THREAD

**Discussion by WebShells** [opensourcepos/opensourcepos#3666] [Team Posts]
Hello everyone, 
Shall we extract a list of changes to be listed in changelog for our new release?
Or almost done?
Not the right time to ask, but any had the chance to check the web and list things to be edited/changed?

**Comment by jekkos** [comment]
there are some existing tools to do this. for example.

https://www.npmjs.com/package/git-release-notes

still one fix to finish on my side.

**Comment by jekkos** [comment]
also you should go through the issues and check what is still open as a bug with milestone 3.3. those should be solved before we can release.
